tests/test_eviction.py

```diff
@@ -120,9 +121,13 @@ def cache_limit_too_low(remote: str, local: str, keep_zip: bool):
     rmtree(local, ignore_errors=False)
 
 
+# TODO: re-enable `shard_eviction` and `cache_limit_too_low` when cache_limit is returned to fully
```

Comment:
Could you let me know if you plan to enable this as part of Delta Table feature support?

Reply:
Almost certainly, but I want to punt these to a future PR in order to maintain velocity.

`cache_limit_too_low` for example relies on `cache_limit` being inviolable. This PR is written with `cache_limit` being briefly violable during decompression and canonicalization, which is more or less unavoidable because we don't know the resulting size of Parquet -> MDS .mds files in advance, and if we end up letting the user intervene in column physical types, which very well will likely be requested down the road if we don't do it as part of this MVP (I cut the relevant arg already), .mds size could be dependent on that, etc